src/ProcessManager/ProcessManager.php

```diff
@@ -0,0 +1,41 @@
+<?php
+
+
+use Broadway\Domain\DomainMessage;
+use Broadway\EventHandling\EventListenerInterface;
+use Illuminate\Foundation\Bus\DispatchesJobs;
+use Madewithlove\LaravelCqrsEs\Inflectors\MethodNameInflector;
+
+abstract class ProcessManager implements EventListenerInterface
+{
+    use DispatchesJobs;
```

Comment:
I don't think we should make this the default. It's possible the user will use another commands bus (like tactician)